osmosis-1/osmosis.zone.json

```diff
@@ -293,6 +293,13 @@
       "osmosis_frontier": true,
       "osmosis_info": true
     },
+    {
+      "chain_name": "uptick",
```

Comment:
We require that osmosis-uptick IBC connection be registered to the Chain Registry. Could you please add there?